Source/Lib/Codec/EbUtility.c

```diff
@@ -128,7 +128,7 @@ PaBlockStats  * pa_get_block_stats(int block_index)
  *  Leading Zeros (NLZ) algorithm to get
  *  the log2f of a 64-bit number
  *****************************************/
-inline uint64_t log2f64(uint64_t x)
+inline uint64_t log2f64adaption(uint64_t x)
 {
```

Comment:
why the change here?

Reply:
Note: Linux build was reporting a conflicting error between log2f64 function in EbUtility.h and <math.h> function with the same name, which seemed unrelated to the code changes being made (details below). Renamed the EbUtility.h function to log2f64adapation and build was successful.

Reply:
How it is related to adding -lp and -ss logic?